src/main/java/lotto/domain/lotto/LottoTicket.java

```diff
@@ -1,30 +1,36 @@
 package lotto.domain.lotto;
 
-import java.util.Collections;
-import java.util.List;
-import java.util.Objects;
+import java.util.*;
 import java.util.stream.Collectors;
 import java.util.stream.IntStream;
+import java.util.stream.Stream;
 
 public class LottoTicket {
+    public static final String SEPARATOR = ",";
 
     private List<LottoNumbers> lottoTicket;
 
     private LottoTicket(int quantity) {
-        checkQuantity(quantity);
-        this.lottoTicket = addLottoNumbers(quantity);
+        this(quantity, Collections.emptyList());
     }
 
-    private LottoTicket(List<LottoNumbers> lottoNumbers) {
-        this.lottoTicket = lottoNumbers;
+    private LottoTicket(int quantity, List<String> enteredManualNumber) {
```

Comment:
구입한 수동 로또가 여러 개 일 수 있으니
```suggestion
    private LottoTicket(int quantity, List<String> enteredManualNumbers) {
```
가 어떨까요?